src/metrics/mongoose_metrics_definitions.hrl

```diff
@@ -82,8 +82,15 @@
           {function, mongoose_metrics, get_mnesia_running_db_nodes_count, [],
            tagged, [value]}},
          {tcpPortsUsed,
-          {function, mongoose_metrics, tcp_ports, [],
-           tagged, [value]}}
+          {probe,
+           [{sampling, fun metrics_probes:do_sample_tcp_connections/0},
+            {datapoints, [value]},
+            {sample_interval, timer:seconds(30)}]}},
+         {processQueueLengths,
+          {probe,
+           [{sampling, fun metrics_probes:do_sample_queue_lengths/0},
+            {datapoints, [regular, fsm, total]},
```

Comment:
IMO it's not the place to define datapoints because they are hardcoded in the actual sampling implementations anyway.